src/components/PricingCard.js

```diff
@@ -1,65 +1,103 @@
+import Button from 'components/Button'
 
-const PricingCard = styled.div`
-  border: 2px solid ${p => p.$color || get('colors.dodgerBlue')};
+const PricingCardWrapper = styled.div`
+  background-color: ${get('colors.valhalla.600')};
   border-radius: 16px;
-  overflow: hidden;
+  padding: 16px;
   height: 100%;
   display: flex;
   flex-direction: column;
+  justify-content: space-between;
+  align-items: flex-start;
+  @media (min-width: ${get('breakpoints.lg')}) {
+    padding: 24px;
+  }
+  @media (min-width: ${get('breakpoints.xl')}) {
+    padding: 26px 32px 40px;
+  }
 `
 
-const PricingCardHeaderWrapper = styled.div`
-  background-color: ${p => p.$color || get('colors.dodgerBlue')};
-  color: ${get('colors.white')};
-  padding: 16px;
-  position: relative;
-  overflow: hidden;
-  min-height: 164px;
-  display: flex;
-  align-items: center;
-  justify-content: center;
+const PlanIcon = styled.div`
+  background-color: ${hexToRgb(get('colors.hotPink'), 0.2)};
+  width: 36px;
+  height: 36px;
+  border-radius: 50%;
+  color: ${get('colors.hotPink')};
+  padding: 8px;
 `
 
-const Decorator = styled.div`
-  background: url(images/decorator6.svg) no-repeat;
-  background-size: 209px 180px;
-  position: absolute;
-  top: -11px;
-  right: -9px;
-  width: 209px;
-  height: 180px;
-  z-index: 1;
+const Title = styled(Typography)`
+  color: ${get('colors.hotPink')};
+  margin-top: 4px;
 `
 
-const HeaderContent = styled.div`
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  flex-direction: column;
-  z-index: 2;
-  position: relative;
-  text-align: center;
+const Pricing = styled(Typography)`
+  color: ${get('colors.ashes.900')};
+  display: block;
+
+  strong {
+    color: white;
+    font-family: 'Poppins';
+    font-size: 26px;
+    line-height: 140%;
+  }
 `
 
-const PricingCardHeader = ({ children, hasDecorator = true, ...props }) => (
-  <PricingCardHeaderWrapper {...props}>
-    <HeaderContent>{children}</HeaderContent>
-    {hasDecorator && <Decorator className="decorator" />}
-  </PricingCardHeaderWrapper>
-)
+const Specification = styled(Typography)`
+  display: block;
+`
 
-const PricingCardBody = styled.div`
-  color: ${get('colors.ashes.100')};
-  background-color: ${p => p.$color || get('colors.valhalla')};
-  height: 100%;
-  display: flex;
-  justify-content: space-between;
-  flex-direction: column;
+const Specifications = styled.div`
+  color: ${get('colors.ashes.600')};
+  margin-top: 16px;
+  @media (min-width: ${get('breakpoints.xl')}) {
+    margin-top: 30px;
+  }
+
+  ${Specification} + ${Specification} {
+    margin-top: 7px;
+    @media (min-width: ${get('breakpoints.xl')}) {
+      margin-top: 12px;
+    }
+  }
 `
 
-PricingCard.Header = PricingCardHeader
-PricingCard.Body = PricingCardBody
+const Cta = styled(Button)`
+  margin-top: 16px;
+  @media (min-width: ${get('breakpoints.xl')}) {
+    margin-top: 32px;
+  }
+`
 
+const PricingCard = ({ plan, ...props }) => {
```

Comment:
Do we need to deconstruct `props` here?

Reply:
Yes, it means that any other props will go to this `props` object, and then I spread it below :)